### PR TITLE
CASMINST-4879: Linting

### PR DESCRIPTION
**Pull request by mharding-hpe**
# Description

Just some linting fun.

# Checklist Before Merging

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/kyverno/Kyverno.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 [Kyverno](https://kyverno.io/) is a policy engine designed specifically for Kubernetes.
 
-Kyverno allows cluster administrators to manage environment specific configurations independently of workload configurations and enforce configuration best practices for their clusters.
+Kyverno allows cluster administrators to manage environment-specific configurations (independent of workload configurations) and enforce configuration best practices for their clusters.
```
Comment (mharding-hpe) [inline operations/kyverno/Kyverno.md:5]:
```suggestion
Kyverno allows cluster administrators to manage environment-specific configurations (independently of workload configurations) and enforce configuration best practices for their clusters.
```

File: operations/kyverno/Kyverno.md [hunk 2]
```diff
@@ -28,19 +28,19 @@ securityContext:
   runAsGroup: 65534
 ```
 
-Currently, [mutation](#mutation) and [validation](#validation) policies are enforced for the network services such as load balancer and virtual service.
+[Mutation](#mutation) and [validation](#validation) policies are enforced for the network services such as load balancer and virtual service.
```
Comment (mharding-hpe) [inline operations/kyverno/Kyverno.md:31]:
```suggestion
[Mutation](#mutation) and [Validation](#validation) policies are enforced for the network services such as load balancer and virtual service.
```

File: operations/kyverno/Kyverno.md [hunk 3]
```diff
@@ -154,52 +151,46 @@ It mutates the manifest of respective workloads before creating it so that when
...
+        ----     ------            ----                           ----               -------
+        Normal   Scheduled         <invalid>                      default-scheduler  Successfully assigned default/nginx to ncn-w003-b7534262
+        Warning  DNSConfigForming  <invalid> (x9 over <invalid>)  kubelet            Search Line limits were exceeded, some search paths have been omitted, the applied search line is: default.svc.cluster.local svc.cluster.local cluster.local vshasta.io us-central1-b.c.vsha-sri-ram-35682334251634485.internal c.vsha-sri-ram-35682334251634485.internal
+        Normal   Pulled            <invalid> (x8 over <invalid>)  kubelet            Container image "nginx:1.14.2" already present on machine
+        Warning  Failed            <invalid> (x8 over <invalid>)  kubelet            Error: container has runAsNonRoot and image will run as root (pod: "nginx_default(0ea1d573-219a-4927-b3c3-c76150d35a7a)", container: nginx)
+        ```
 
-6. (`ncn#`) If the previous step failed, delete the policy and restart the workload.
+1. (`ncn-mw#`) If the previous step failed, delete the policy and restart the workload.
```
Comment (mharding-hpe) [inline operations/kyverno/Kyverno.md:187]:
```suggestion
1. (`ncn-mw#`) If the previous step failed, then delete the policy and restart the workload.
```

**Review by rustydb — APPROVED** [review]

File: operations/kyverno/Kyverno.md [hunk 4]
```diff
@@ -252,57 +243,55 @@ Also, it generates the report of policy violation in respective workloads. The f
...
     ```
 
-    (`ncn#`) View the policy report status with the following command:
-
-    ```bash
-    kubectl get polr -A
-    ```
-
-    Example output:
+    * (`ncn-mw#`) View the policy report status with the following command:
```
Comment (rustydb) [inline operations/kyverno/Kyverno.md:246]: Should this be a "-'?

**Review by rustydb — APPROVED** [review]

File: operations/kyverno/Kyverno.md (hunk 4 above)
Comment (rustydb) [reply operations/kyverno/Kyverno.md:246]: I don't see the linter complaining, maybe it doesn't see it or was the linter updated? I'm mobile so it's a little tedious to check.

**Review by rustydb — APPROVED** [review]

File: install/deploy_non-compute_nodes.md [hunk 5]
```diff
@@ -218,7 +218,7 @@ for all nodes, the Ceph storage will have been initialized and the Kubernetes cl
 
     > **NOTES:**
     >
-    > - If the nodes have PXE boot issues (e.g. getting PXE errors, not pulling the ipxe.efi binary) then see [PXE boot troubleshooting](troubleshooting_pxe_boot.md).
+    > - If the nodes have PXE boot issues (e.g. getting PXE errors, not pulling the `ipxe.efi` binary), then see [PXE boot troubleshooting](troubleshooting_pxe_boot.md).
```
Comment (mharding-hpe) [inline install/deploy_non-compute_nodes.md:221]:
```suggestion
    > - If the nodes have PXE boot issues (e.g. getting PXE errors, not pulling the `ipxe.efi` binary), then see [Troubleshooting PXE Boot](troubleshooting_pxe_boot.md).
```

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**